ajax/libs/geomicons-open/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "geomicons-open",
+  "filename": "geomicons.min.js",
+  "version": "2.0.0",
+  "description": "Open Source Icons for the Web",
+  "homepage": "http://geomicons.com/",
```

Comment:
@sashberd I can't reach this site. Can you?

Reply:
0.0.1 is exists on npm